src/components/players/list/index.js

```diff
@@ -65,9 +61,12 @@ class List extends React.Component {
     };
 
     render() {
+        const { isLoading } = this.props.selectedTeam;
+
         return (
-            <div ref={this.ref} className={styles.playersListContainer}>
+            <div className={styles.container}>
                 {this.renderPlayers()}
+                {isLoading && <Overlay isLoading={isLoading} />}
```

Comment:
Move this above, Line 68. In addition read my comment about the suggested change to the overlay